stage0_bin/src/asm/boot.s

```diff
@@ -91,7 +91,7 @@ _protected_mode_start:
     and $0b100, %eax          # eax &= 0b100; -- SEV-SNP active
     test %eax, %eax           # is eax zero?
     je 2f                     # if yes, no SNP, skip validation and jump ahead
-    mov $0x1000, %ebx         # ebx = 0x1000 -- start address (skipping first page as that's not mapped)
+    mov $0x0000, %ebx         # ebx = 0x0000 -- start address
```

Comment:
Could you add a comment that this is needed specifically for Linux, even though in the Restricted Kernel we don't care? It may save future us some headaches.

Reply:
Honestly I was surpised that Restricted Kernel didn't seem to care. Plus, I'd argue the existing behaviour (of skipping the first 4K of memory) is the surprising one that would require a comment.

Reply:
If someone at some point thought this logic made sense, I would still err on the side of more comments. Anyways you and Conrad are the experts.